ale/drivers/base.py

```diff
@@ -458,6 +468,18 @@ def number_of_quaternions(self):
         # always one for framers
         return 1
 
+    @property
+    def center_ephemeris_time(self):
+        """
+        The center ephemeris time for a framer.
+        """
+        center_time = self.starting_ephemeris_time + self.exposure_duration / 2
+        return center_time
+
+    @property
+    def exposure_duration(self):
+        return self._exposure_duration.value * 0.001  # Scale to seconds
```

Comment:
I would just store the value and scale it wherever `self._exposure_duration` is being set.